packages/roosterjs-editor-plugins/lib/TabIndent/TabIndent.ts

```diff
@@ -113,4 +144,34 @@ export default class TabIndent implements EditorPlugin {
 
         return false;
     }
+
+    // Check if it is a blockquote event, if it is true, also return the blockquote element where the cursor resides
+    // 1. Cursor is in blockquote element
+    // 2. Current block has no content
+    // 3. is keyDown
+    // 4. is Enter
+    // 5. Any of ctrl/meta/alt is not pressed
+    private isBlockQuoteEvent(event: PluginEvent, interestedKeyCodes: number[], nodeAtCursor: Node): [boolean, Node] {
+        let isBlockQuoteEvent = false;
+        let blockQuoteElement: Node;
+
+        if (event.eventType == PluginEventType.KeyDown) {
+            let keybordEvent = (event as PluginDomEvent).rawEvent as KeyboardEvent;
+            blockQuoteElement = getBlockQuoteElement(this.editor, nodeAtCursor);
+            let contentTraverser = this.editor.getContentTraverser(ContentScope.Selection);
+            let blockElement = !!contentTraverser ? contentTraverser.currentBlockElement : null;
+
+            // Get the content of the current block element, remove any zero white spaces
+            let content = !!blockElement ? blockElement.getTextContent().replace(/\u200B/g, '') : null;
+
+            isBlockQuoteEvent = !!blockQuoteElement &&
+                !content &&
+                interestedKeyCodes.indexOf(keybordEvent.which) >= 0 &&
```

Comment:
This function need to be fail fast, so do the key check before all other checks.

Reply:
Fixed